stream_alert_cli/test.py

```diff
@@ -270,7 +278,31 @@ def _run_rule_tests(self, file_name, test_event, formatted_record, print_header_
             if message:
                 self.status_messages.append(StatusMessage(StatusMessage.FAILURE, message))
         elif not alerted_properly:
-            message = 'Rule failure: [{}.json] {}'.format(file_name, test_event['description'])
+            message = ('Test failure: [{}.json] Test event with description '
+                       '\'{}\'').format(file_name, test_event['description'])
+            if alerts and not test_event['trigger_rules']:
+                # If there was a failure due to alerts triggering for a test event
+                # that does not have any trigger_rules configured
+                context = 'is triggering the following rules but should not be: {}'
+                trigger_rules = ' ,'.join('\'{}\''.format(alert['rule_name']) for alert in alerts)
+                message = '{} {}'.format(message, context.format(trigger_rules))
+            elif bad_alerts:
+                # If there was a failure due to alerts triggering for other rules outside
+                # of the rules defined in the trigger_rules list for the event
+                context = 'is triggering the following rules but should not be: {}'
```

Comment:
Would prefer to differentiate the failure here is different from above by having a different context.

Reply:
Just squashed - this should be different than the other one now.